Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py

```diff
@@ -0,0 +1,664 @@
+    }
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def is_deceptive_server_command(client: Client, args: dict) -> Tuple:
+    hostname = args.get("hostname", None)
+    is_deceptive_server = True
+    r = client.is_deceptive_server(hostname)
+    if not r:
+        is_deceptive_server = False
+
+    result = {
+        'Hostname': hostname,
+        'IsDeceptiveServer': is_deceptive_server
+    }
+    readable_output = tableToMarkdown('Illusive Is Deceptive', result)
+    outputs = {
+        'Illusive.IsDeceptive(val.Hostname == obj.Hostname)': result
+    }
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def delete_deceptive_users_command(client: Client, args: dict) -> Tuple:
+    deceptive_users = argToList(args.get('deceptive_users'))
+
+    client.delete_deceptive_users(deceptive_users)
+    if len(deceptive_users) > 1:
+        a, b = "s", "were"
+    else:
+        a, b = "", "was"
+    result = f' {"Deceptive User{} {} {} successfully Deleted".format(a, deceptive_users, b)}'
+    readable_output = f'## {result}'
+
+    outputs: Dict[str, Any] = {
+    }
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def delete_deceptive_servers_command(client: Client, args: dict) -> Tuple:
+    deceptive_servers = argToList(args.get('deceptive_hosts'))
+
+    client.delete_deceptive_servers(deceptive_servers)
+    if len(deceptive_servers) > 1:
+        a, b = "s", "were"
+    else:
+        a, b = "", "was"
+    result = f' {"Deceptive Server{} {} {} successfully Deleted".format(a, deceptive_servers, b)}'
+    readable_output = f'## {result}'
+
+    outputs: Dict[str, Any] = {
+    }
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def get_incidents_command(client: Client, args: dict) -> Tuple:
+    incident_id = args.get("incident_id")
+    has_forensics = args.get("has_forensics", None)
+    host_names = argToList(args.get('host_names'))
+    limit = args.get("limit", 10)
+    offset = args.get("offset", 0)
+    start_date = args.get("start_date", None)
+    if incident_id:
+        incident = client.get_incident(incident_id)
+    else:
+        limit = "100" if int(limit) > 100 else limit
+        incident = client.list_all_incidents(has_forensics, host_names, limit, offset, start_date)
+
+    readable_output = tableToMarkdown('Illusive Incidents', incident)
+    outputs = {
+        'Illusive.Incident(val.incidentId == obj.incidentId)': incident
+    }
+    return (
+        readable_output,
+        outputs,
+        incident  # raw response - the original response
+    )
+
+
+def get_event_incident_id_command(client: Client, args: dict) -> Tuple:
+    event_id = argToList(args.get("event_id"))
```

Comment:
event_id is a list? why?

Reply:
because of builtin polling function of Demisto, there is no other way..